cli/ballerina-packerina/src/test/java/org/ballerinalang/packerina/cmd/BuildCommandTest.java

```diff
@@ -804,9 +804,33 @@ public void testBuildWithSkipTests() throws IOException {
                 "\nGenerating executables\n" +
                 "\ttarget/bin/module1.jar\n");
     }
-    
-    // Check compile command inside a module directory
 
+    @Test(description = "Test the project with Maven dependencies")
+    public void testBuildWithMavenDependencies() throws IOException {
+        // valid source root path
+        Path validBalFilePath = this.testResources.resolve("project-with-maven-dependencies");
+        BuildCommand buildCommand = new BuildCommand(validBalFilePath, printStream, printStream, false, true);
+        new CommandLine(buildCommand).parse("-a");
+        buildCommand.execute();
+        Path target = validBalFilePath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
+        Path platformLibsDir = target.resolve("platform-libs");
+        Assert.assertTrue(Files.exists(target), "Check if target directory is created");
+        Assert.assertTrue(Files.exists(platformLibsDir), "Check if platform-libs is created");
+
+        //Assert maven dependencies are downloaded
+        Assert.assertTrue(Files.exists(Paths.get(getJarPath(platformLibsDir.toString(), "org.apache.maven",
+                "maven-artifact", "3.6.3"))));
+        Assert.assertTrue(Files.exists(Paths.get(getJarPath(platformLibsDir.toString(), "org.ballerinalang",
+                "wso2-cassandra", "0.8.3"))));
+
+    }
+
```

Comment:
Shall we add a test case to check the expected behavior when only the group `[[platform.repositories]]` is given without any entries?

Reply:
Added tests in 65d2559295475f7c7ef375e674939046165b74fc